src/how_to/experiment.md

```diff
@@ -13,16 +12,21 @@ When doing an experiment, there are two roles:
 
 These can be the same person, but in that case, you should find someone else to second the PR.
 
+
+
 ## Process
 
-The process is as follows:
+If you are an experienced Rust contributor who would like to start an experiment in-tree, the process is as follows:
 
-* The **experimentor** should open a PR adding a new feature gate to the compiler, along with an associated tracking issue. They should include a short write-up documenting the motivation and outline of what they are trying to achieve. 
-* The **lang-team champion** can "second" the PR, starting an FCP. Once the FCP completes, the PR can land and implementation work begins (always gated under the new feature gate).
+* Write-up a description of the problem you are trying to solve and the general shape of the solution you want to work on. Discuss it on Zulip or elsewhere to find a **lang-team champion**:
+    * The champion is the connection to the lang-team. They can check in with you from time to time to see how the work is going and relay those updates to the lang-team (of course, you're always welcome to join meetings yourself too!). They can also help to discuss problems that arise.
+* Once you've found a champion, open a PR adding a new feature gate to the compile and create an associated tracking issue.
+    * The PR should include a write-up documenting the motivation and outline of what they are trying to achieve. 
```

Comment:
```suggestion
    * The PR and tracking issue should include a write-up documenting the motivation and outline of what they are trying to achieve. 
```

(In general feels like a good idea to preserve it a little more durably than PR descriptions).